distribution/archives/tar/build.gradle

```diff
@@ -1,33 +1 @@
-/*
- * Licensed to Elasticsearch under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-task buildTar(type: Tar) {
-  dependsOn createLogDir, createPluginsDir
-  baseName = 'elasticsearch'
-  extension = 'tar.gz'
-  with archivesFiles
-  compression = Compression.GZIP
-  dirMode 0755
-  fileMode 0644
-}
-
-artifacts {
-  'default' buildTar
-}
 
```

Comment:
We should probably keep the license file and a comment explaining that the build is handled by the parent.